src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -55,11 +52,8 @@ protected ExportProvider ExportProvider
         protected virtual TestComposition GetComposition()
             => s_baseComposition.AddParts(GetCompletionProviderType());
 
-        public override void Dispose()
-        {
-            this.WorkspaceFixture.DisposeAfterTest();
```

Comment:
📝 The `IDisposable.Dispose` method runs _after_ the `UseExportProviderAttribute.After` method, which means it is not capable of disposing a workspace instance used by a test. All impacted tests that I found have been updated to dispose of the workspace in the correct order.